pdftotree/core.py

```diff
@@ -66,8 +65,8 @@ def parse(
         log.info("Tree structure built, creating html...")
         pdf_html = extractor.get_html_tree()
         log.info("HTML created.")
-        # Check html_path exists, create if not
-        pdf_html = re.sub(r"[\x00-\x1F]+", "", pdf_html)
+        # TODO: what is the following substition for and is it required?
```

Comment:
If I recall, we had some filenames with special characters that caused issues, so this was taking the first 31 special ASCII chars and turning them into nothing.

I don't know if it's is important to do anymore though, and it seems fine to leave it out until someone runs into it again (if someone runs into it again).

Reply:
filenames? A bit strange because this line of code substitutes characters in the content.
Anyway, I'll keep this line commented out until this causes some issue again.
Thanks for your recall!